2-mandatory/6-journey-planner.js

```diff
@@ -121,9 +121,10 @@ function getLocationName() {}
 
   Advanced challange: try to use arrow function when invoking an array method.
 */
-function journeyPlanner(locations, transportMode) {
-  // Implement the function body
-}
+const journeyPlanner = (locations, transportMode) =>
+  locations
+    .filter((element) => isAccessibleByTransportMode(element, transportMode))
+    .map(getLocationName);
```

Comment:
These are all so good Laleh! I also LOVE how you are chaining your functions (.filter and .map) on different lines!!! This is a great style of coding, and makes things super readable! My senior developer at my company would offer you a job just on this styling alone! Great work! 🎉 💃 👏

Reply:
oh, thank you so much :)